plugins/tedge_apt_plugin/tests/main.rs

```diff
@@ -1,163 +1,172 @@
+    // input command
+    //
+    // expected stderr
+    //
+    // expected exit code
+    //
+    // description about the test
+    #[test_case(
     &format!("install {} --file {}", PACKAGE_NAME, "wrong_path"),
     "ERROR: Parsing Debian package failed",
     5
     ; "wrong path"
 )]
-#[test_case(
+    #[test_case(
     &format!("install {} --file {} --module-version {}", PACKAGE_NAME, "not/a/package/path", PACKAGE_VERSION),
     "ERROR: Parsing Debian package failed",
     5
     ; "wrong path with right version"
 )]
-#[test_case(
+    #[test_case(
     &format!("install {} --file {} --module-version {}", PACKAGE_NAME, PACKAGE_FILE_PATH, "some_version"),
     "ERROR: Parsing Debian package failed",
     5
     ; "right path with wrong version"
 )]
-#[test_case(
+    #[test_case(
     &format!("install {} --file {} --module-version {}", PACKAGE_NAME, "not/a/package/path", "some_version"),
     "ERROR: Parsing Debian package failed",
     5
     ; "wrong path with wrong version"
 )]
-fn install_from_local_file_fail(
-    input_command: &str,
-    expected_stderr: &str,
-    expected_exit_code: i32,
-) -> anyhow::Result<()> {
-    // no setup needed, wrong arguments are provided to tedge apt plugin
-    let (stdout, stderr, exit_code) = run_cmd(TEDGE_APT_COMMAND, input_command)?;
-
-    // asserting command failed
-    assert_that!(stdout.is_empty(), true);
-    assert_that!(stderr, match_regex(expected_stderr));
-    assert_that!(exit_code, eq(expected_exit_code));
-    Ok(())
-}
-
-// Parameters:
-//
-// input command
-//
-// expected stderr
-//
-// expected exit code
-//
-// description about the test
-#[test_case(
+    fn install_from_local_file_fail(
+        input_command: &str,
+        expected_stderr: &str,
+        expected_exit_code: i32,
+    ) -> anyhow::Result<()> {
+        // no setup needed, wrong arguments are provided to tedge apt plugin
+        let (stdout, stderr, exit_code) = run_cmd(TEDGE_APT_COMMAND, input_command)?;
+
+        // asserting command failed
+        assert_that!(stdout.is_empty(), true);
+        assert_that!(stderr, match_regex(expected_stderr));
+        assert_that!(exit_code, eq(expected_exit_code));
+        Ok(())
+    }
+
+    // Parameters:
+    //
+    // input command
+    //
+    // expected stderr
+    //
+    // expected exit code
+    //
+    // description about the test
+    #[cfg(feature = "requires-sudo")]
+    #[test_case(
     &format!("install {} --file {}", PACKAGE_NAME, PACKAGE_FILE_PATH),
     PACKAGE_NAME,
     0
     ; "path"
 )]
-#[serial]
-#[test_case(
+    #[serial]
+    #[test_case(
     &format!("install {} --file {} --module-version {}", PACKAGE_NAME, PACKAGE_FILE_PATH, PACKAGE_VERSION),
     PACKAGE_NAME,
     0
     ; "path with version"
 )]
-#[serial]
-#[test_case(
+    #[serial]
+    #[test_case(
     &format!("install {} --module-version {} --file {}", PACKAGE_NAME,  PACKAGE_VERSION, PACKAGE_FILE_PATH),
     PACKAGE_NAME,
     0
     ; "version with path"
 )]
```

Comment:
shouldn't this be indented as well (same for the above instances of `test_case()`)?